### ISSUE TITLE
Website Dark Theme Text Unreadable Webkit

### ISSUE THREAD

**Issue by in0rdr** [opentofu/opentofu#802]
### OpenTofu Version

```shell
n/a
```

### OpenTofu Configuration Files

n/a

### Debug Output

n/a

### Expected Behavior

The readability of the text on the website is good (has good contrast), so that I can read the docs.

### Actual Behavior

The readability of the website/docs is bad on some lesser known webkit browser (as Falkon). The contrast of the font is bad, dark font on dark background:

![2023-10-31-080625_1568x684_scrot](https://github.com/opentofu/opentofu/assets/9302430/ca6c5f31-4bdd-46ca-821c-84f71c955a11)

This might be a "me problem" though :)

### Steps to Reproduce

Browse the website with Falkon 23.04.3

### Additional Context

I noticed that the dark theme is the default. Also, I noticed that the Falkon browser has no issues with rendering readable text with the light theme. Using the ligth theme by default might already help the accesibility in these lesser known/used webkit browsers.

**Comment by janosdebugs** [comment]
Hey @in0rdr is this still an issue? I'm picking up website and docs issues and I was going through the backlog of things that need doing.

**Comment by in0rdr** [comment]
@janosdebugs thanks for the follow-up. Yes, seems this is still an issue in the said browser. I just did a quick check.

**Comment by janosdebugs** [comment]
Thanks for the feedback @in0rdr, I'll provisionally assign this to myself. If nobody else fixes it, I'll install Falkon and fix it within the next few weeks.

**Comment by janosdebugs** [comment]
Hey @in0rdr I checked the OpenTofu website and it indeed looks broken. I have also checked a few additional websites such as [kubernetes.io](https://kubernetes.io) or the [Google Cloud front page](https://cloud.google.com/) and it seems like they are all somewhat broken. It seems to me like the Falkon browser doesn't support CSS features that are today considered universal, such as `calc()` which is [available in 98%+ of browsers](https://caniuse.com/?search=calc).

![Screenshot of the Falkon browser indicating an unsupported value.](https://github.com/opentofu/opentofu/assets/86970079/18e1d0f4-e4ff-4122-8fd7-0362ee001b14)

This seems like this is something Falkon should fix or add support for. Unfortunately, working around this on our side would require rebuilding large parts of the theme we are using, for which we currently don't have capacity.
